diff --git a/model_filter_type.go b/model_filter_type.go
index dbf4745..33d9659 100644
--- a/model_filter_type.go
+++ b/model_filter_type.go
@@ -26,10 +26,10 @@ const (
 	MATCH_PREFIX FilterType = "MatchPrefix"
 	MATCH_SUFFIX FilterType = "MatchSuffix"
 	MATCH_INFIX FilterType = "MatchInfix"
-	LESS_THAN FilterType = "LessThan"
-	LESS_THAN_OR_EQUAL FilterType = "LessThanOrEqual"
-	GREATER_THAN FilterType = "GreaterThan"
-	GREATER_THAN_OR_EQUAL FilterType = "GreaterThanOrEqual"
+	FT_LESS_THAN FilterType = "LessThan"
+	FT_LESS_THAN_OR_EQUAL FilterType = "LessThanOrEqual"
+	FT_GREATER_THAN FilterType = "GreaterThan"
+	FT_GREATER_THAN_OR_EQUAL FilterType = "GreaterThanOrEqual"
 )
 
 // All allowed values of FilterType enum
diff --git a/model_segment_condition.go b/model_segment_condition.go
index 18f0d2c..b023ae0 100644
--- a/model_segment_condition.go
+++ b/model_segment_condition.go
@@ -20,12 +20,12 @@ type SegmentCondition string
 
 // List of SegmentCondition
 const (
-	GREATER_THAN SegmentCondition = "GreaterThan"
-	GREATER_THAN_OR_EQUAL SegmentCondition = "GreaterThanOrEqual"
-	LESS_THAN SegmentCondition = "LessThan"
-	LESS_THAN_OR_EQUAL SegmentCondition = "LessThanOrEqual"
-	EQUAL SegmentCondition = "Equal"
-	UNDEFINED SegmentCondition = "Undefined"
+	SC_GREATER_THAN SegmentCondition = "GreaterThan"
+	SC_GREATER_THAN_OR_EQUAL SegmentCondition = "GreaterThanOrEqual"
+	SC_LESS_THAN SegmentCondition = "LessThan"
+	SC_LESS_THAN_OR_EQUAL SegmentCondition = "LessThanOrEqual"
+	SC_EQUAL SegmentCondition = "Equal"
+	SC_UNDEFINED SegmentCondition = "Undefined"
 )
 
 // All allowed values of SegmentCondition enum
